.github/workflows/build.yml

```diff
@@ -0,0 +1,108 @@
+name: build-pipeline
+on:
+  pull_request:
+    branches: [master]
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - remove-vend
+jobs:
+  pre-checks:
+    runs-on: ubuntu-latest
+    steps:
+      # Install golang
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+
+      # Checkout to the latest commit
+      # On specific directory/path
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: gofmt check
+        run: |
+          if [ "$(gofmt -s -l . | wc -l)" -ne 0 ]
+          then
+           echo "The following files were found to be not go formatted:"
+           gofmt -s -l .
+           exit 1
+          fi
+
+      - name: golangci-lint
+        uses: reviewdog/action-golangci-lint@v1
+
+      - name: unused-package check
+        run: |
+          make unused-package-check
+
+  image-build:
+    runs-on: ubuntu-latest
+    needs: pre-checks
+    steps:
+      # Checkout to the latest commit
+      # On specific directory/path
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+
+      - name: Build Docker Image
+        env:
+          DOCKER_REPO: litmuschaos
+          DOCKER_IMAGE: chaos-operator
+          DOCKER_TAG: ci
+        run: |
+          make build-chaos-operator
+
+  tests:
+    runs-on: ubuntu-latest
+    needs: pre-checks
+    steps:
+      # Install golang
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+
+      # Checkout to the latest commit
+      # On specific directory/path
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      #Install and configure a kind cluster
+      - name: Installing Prerequisites (KinD Cluster)
+        uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.7.0"
+
+      - name: Configuring and testing the Installation
+        run: |
+          kubectl cluster-info --context kind-kind
+          kind get kubeconfig --internal >$HOME/.kube/config
+          kubectl get nodes
+
+      - name: Dependency checks
+        run: |
+          make deps
+
+      - name: Running Go BDD Test
+        run: |
+          make test
+
+  security-scan:
+    runs-on: ubuntu-latest
+    needs: pre-checks
+    steps:
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
```

Comment:
Can we use a release version `aquasecurity/trivy-action@master`?

Reply:
Earlier, we used the release version in the chaos-exporter. But it was failing due to some reason.

Reply:
older release if possible otherwise it is okay, but it can break anytime?